HW0/P6/P6B.py

```diff
@@ -16,16 +16,22 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(x/10.0) for x in range(-60,1,1)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
-
+        parallelStart = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
Despite the notation being very confusing, map can actually be used in this way.  If you test the code, it is as if I fed in [t,t,t,t,t,t,...,t].  I shouldn't use map this way in the future.  It is not clear code.  Thanks for pointing it out.